TicketSystem/TicketSystem/Program.cs

```diff
@@ -34,4 +34,6 @@
 app.UseAuthorization();
 app.MapControllers();
 
-app.Run();
\ No newline at end of file
+app.Run();
+
+public partial class Program { }
```

Comment:
Not as an issue, just another possible way to make a Program.cs accessible for the test project is added to your TicketSystem.csproj something like:

	<ItemGroup>
		<InternalsVisibleTo Include="IVCRM.API.IntegrationTests" />
	</ItemGroup>